### ISSUE TITLE
No update on repos but update in playstore

### ISSUE THREAD

**Issue by xunil89** [cylonid/NativeAlphaForAndroid#54]
As per title I noticed an app update with buxfixes on the playstore but I don't see any update on the ghitub repos. Has the app become closed source? Or is the app on the playstore managed by a third party and is a fork?

**Comment by cylonid** [comment]
No, it's neither closed source nor is the PlayStore app a fork. The release here and in consequence in IzzyOnDroid/F-Droid will follow soon.
